classes/apirest/apirest.php

```diff
@@ -238,9 +238,9 @@ public function create_evidence_item($evidenceitem, $credentialid, $throwerror =
     public function create_evidence_item_duration($startdate, $enddate, $credentialid, $hidden = false) {
 
         $durationinfo = array(
-            'start_date' => date("Y-m-d", strtotime($startdate)),
```

Comment:
That's true for now, but `$startdate` is one of the arguments so it could not be a timestamp depending on what value will be passed to this function.

If we'd like to accept only `$startdate` as a timestamp, we should update the PHPDoc as `@param int $starttimestamp` or something and also consider updating `@param string $enddate` for consistency.

Reply:
I have checked the whole plugin looking when this method is called and I can confirm that we always send a timestamp for both `startdate` and `enddate`. I have updated the PHPDoc for the updated methods.